x/pylons/client/cli/tx_execute_recipe_test.go

```diff
@@ -139,7 +140,7 @@ func TestExecuteRecipeNoInputOutput(t *testing.T) {
 
 	// this recipe can be run infinitely
 	// run it 5x in a loop
-	common = CommonArgs(address, net)
+	common = util.CommonArgs(address, net)
 	for i := 0; i < 5; i++ {
```

Comment:
"5" can be in a constant.

Reply:
i didn't write this, but ig it could be numReps or smth? i was just trying not to touch other parts of the codebase more than necessary